tst/EnergyPlus/unit/PurchasedAirManager.unit.cc

```diff
@@ -1400,6 +1422,38 @@ TEST_F(ZoneIdealLoadsTest, IdealLoads_Fix_SA_HumRat_Test)
         "EnergyManagementSystem:Program,",
         "Test_InsideHVACSystemIterationLoop,",
         "set Mdot = 0.1;",
+
+        "Schedule:Constant,",
+        "  DXHeatingCoilFuelEffSched,    !- Name",
+        "  AnyValue,                     !- Schedule Type Limits Name",
+        "  1.0;                          !- Field 1",
+
+        "Schedule:Constant,",
+        "  DXCoolingCoilFuelEffSched,    !- Name",
+        "  AnyValue,                     !- Schedule Type Limits Name",
+        "  1.0;                          !- Field 1",
+
+        "EnergyManagementSystem:Actuator,",
+        "  HeatingFuelEff_SCH_Override,  !- Name",
+        "  DXHeatingCoilFuelEffSched,    !- Actuated Component Unique Name",
+        "  Schedule:Constant,            !- Actuated Component Type",
+        "  Schedule Value;               !- Actuated Component Control Type",
+
+        "EnergyManagementSystem:Actuator,",
+        "  CoolingFuelEff_SCH_Override,  !- Name",
+        "  DXCoolingCoilFuelEffSched,    !- Actuated Component Unique Name",
+        "  Schedule:Constant,            !- Actuated Component Type",
+        "  Schedule Value;               !- Actuated Component Control Type",
+
+        "EnergyManagementSystem:ProgramCallingManager,",
+        "  IdealLoads_Fuel_Eff_Reset_ProgMgr, !- Name",
+        "  InsideHVACSystemIterationLoop,     !- EnergyPlus Model Calling Point",
+        "  IdealLoadsAirSystemFuelEffProg;    !- Program Name 2",
+
+        "EnergyManagementSystem:Program,",
+        "  IdealLoadsAirSystemFuelEffProg,    !- Name",
+        "  Set HeatingFuelEff_SCH_Override = 2.0,  !- Program Line 1",
+        "  Set CoolingFuelEff_SCH_Override = 3.0;  !- Program Line 2",
```

Comment:
This unit test demonstrates the use of EMS to alter fuel efficiency values specified in schedule objects (e.g., 1.0) to 2.0 and 3.0 for heating and cooling efficiencies, respectively. The Schedule actuator overrides the initial values specified in the `ZoneHVAC:IdealLoadsAirSystem` object input fields.